pkg/sidecars/server/handlers.go

```diff
@@ -22,9 +22,8 @@ func (s *Server) writeMessageHandler() rest.Handler {
 	return func(w http.ResponseWriter, r *http.Request) {
 		start := time.Now()
 
-		logger.Info("handling message write")
-
-		channel := strings.TrimPrefix(r.URL.Path, "/")
+		channel := strings.TrimPrefix(r.URL.Path, "/channel/")
```

Comment:
The broker specific sidecar doesnt need to have the /channel/ since it doesnt deal w/ routes. Its not wrong, just unescessary. You can leave it or remove, your call